e-commerce-website/db.json

```diff
@@ -1117,5 +1117,18 @@
       "createdAt": "2021-05-11T13:36:13.145Z",
       "updatedAt": "2021-09-11T09:28:38.332Z"
     }
+  ],
+  "users": [
+    {
+      "accessToken": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjdiODcxMTIzNzU0MjdkNjU3ZjVlMjVjYTAxZDU2NWU1OTJhMjMxZGIiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJodHRwczovL3NlY3VyZXRva2VuLmdvb2dsZS5jb20vcmVhY3QtZS1jb21tZXJjZS1mNjUyZCIsImF1ZCI6InJlYWN0LWUtY29tbWVyY2UtZjY1MmQiLCJhdXRoX3RpbWUiOjE2MzI2NzM5NDgsInVzZXJfaWQiOiJDRDFoSmZVdWd5TFJSQnZjWDJRcHJkZjJJbHYxIiwic3ViIjoiQ0QxaEpmVXVneUxSUkJ2Y1gyUXByZGYySWx2MSIsImlhdCI6MTYzMjY3Mzk0OCwiZXhwIjoxNjMyNjc3NTQ4LCJlbWFpbCI6InRydW5nbGU5OTlAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJmaXJlYmFzZSI6eyJpZGVudGl0aWVzIjp7ImVtYWlsIjpbInRydW5nbGU5OTlAZ21haWwuY29tIl19LCJzaWduX2luX3Byb3ZpZGVyIjoicGFzc3dvcmQifX0.n48Kb0RYR_f5YJlfndpPMS2_--jfJFsOhsNAD6-RqZdtt5-_TXJOKAGOQD6833TBLgzkmb_4TZD4SEbKzTX_S2HX-aoO_kilunAFhJPxVKRuXExwKRf_3cgAIc3nF3FSFjV8BCLtK2WCzNf0So-0Q9s7Be1mw1JKsOGW_G6ecRokcYyU_sCPz1d0DjcnCLHbeL1D9-ooYH9uVNzqzrwnQz2h9xsiZGXeTVx0PW5vFeg2pfVtTmi5hY-l1T7aDypqOu2qLxxnByOPOXZWeWp3uh3r9kxo3pMs8dVrrxKmLXn2yAxoVt7upgpO4x2uI5ucVS919DG_P1TqcbnnPgIb4A",
```

Comment:
Sao lại lưu access token vào database nhỉ
Theo nguyên lí thì token sẽ có một khoảng thời gian hết hạn nhất định
Từ đó nó sẽ tạo 1 token mới hoàn toàn chứ đâu phải 1 token cố định đâu mà lưu trong db